### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by Syn34**
Ratchet & Clank: Rift Apart recently got a PC release and we created a splitter for the game

If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [ x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [ x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x ] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

**Comment by Syn34** [comment]
Not exactly sure what caused this exit error code. The script seems fine and the entry of the html into the branch seems correct?

**Comment by just-ero** [comment]
> Not exactly sure what caused this exit error code. The script seems fine and the entry of the html into the branch seems correct?

You can always check the CI output: https://github.com/LiveSplit/LiveSplit.AutoSplitters/actions/runs/5734383431/job/15540509236?pr=1443#step:8:95

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+    <AutoSplitter>
+        <Games>
+            <Game>Ratchet &amp; Clank: Rift Apart</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/Syn34/Rift-Apart-Autosplitter/main/Riftsplitter%20v13%20by%20Jasper%20%26%20Sin.asl</URL>
+        </URLs>
+        <Type>Script</Type>
+        <Description>Supports Splitting and Load Removal for v1.728.0.0</Description> 
+    </Autosplitter>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:12]:
```suggestion
    </AutoSplitter>
```

**Comment by Syn34** [comment]
Thank you for helping with this !!!

**Pull request merged**